pandas/tests/indexes/multi/test_base.py

```diff
@@ -0,0 +1,938 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
as long as you are splitting, would not create a test_base, rather create functional named files, similar to how ``pandas/tests/indexes/datetimes``; since you already have some functional area files below, just distribute these